test/plugin/base_test.rb

```diff
@@ -1192,6 +1390,34 @@ def container_log_entry(log, stream = CONTAINER_STREAM)
     }
   end
 
+  def gke_application_container_log_entry(log)
+    {
+      log: log,
+      LOCAL_RESOURCE_ID_KEY =>
+        "gke_containerName.#{CONTAINER_NAMESPACE_ID}" \
+        ".#{CONTAINER_POD_NAME}.#{CONTAINER_CONTAINER_NAME}"
+    }
+  end
+
+  # Docker Container
+  def docker_container_stdout_stderr_log_entry(
```

Comment:
[Optional] `docker_container_stream_log_entry`...

Reply:
Hmm... I think `stderr` and `stdout` is more straight forward.

Reply:
Fine, but can we at least be consistent about the ordering of `stdout_stderr` vs `stderr_stdout`?

Reply:
Makes sense. I changed all occurrences to `stdout_stderr`.